monai/networks/nets/cell_sam_wrapper.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import torch
+from torch import nn
+from torch.nn import functional as F
+
+try:
+    from segment_anything.build_sam import build_sam_vit_b
```

Comment:
Please use `optional_import` to have optional packages included in code. If this package isn't present nothing should be said or raised, but the class should raise an exception upon instantiation.

Reply:
ok I changed it, and added a unit test. plz check. thank you!